apps/web/src/components/MenuBar/MenuBar.tsx

```diff
@@ -31,6 +31,23 @@ const useStyles = makeStyles((theme: Theme) =>
       [theme.breakpoints.down('sm')]: {
         height: `${theme.mobileFooterHeight}px`,
         padding: 0,
+        '& .MuiButton-textLabel': {
+          display: 'none',
+        },
+        '& .MuiButton-mobileBackground': {
+          marginRight: '1em',
```

Comment:
```suggestion
          margin: '0 0.5em',
```
This will cause the buttons to be perfectly centered. With marginRight, it's a little lopsided. There's a little more white space on the right than on the left:

![image](https://user-images.githubusercontent.com/12685223/176973328-0fe60964-a994-4977-8441-73bf1cdaf619.png)